### PR TITLE
Add open action for android

### PR DESCRIPTION
**Pull request by xiushudongfang**
Open games from file explorer directly.

### CODE REVIEW

File: android/AndroidManifest.xml [hunk 1]
```diff
@@ -47,6 +47,63 @@
...
+					android:scheme="file" />
+				<data
+					android:host="*"
+					android:mimeType="*/*"
+					android:pathPattern=".*\\.cso"
+					android:scheme="file" />
+				<data
+					android:host="*"
+					android:mimeType="*/*"
+					android:pathPattern=".*\\.isz"
```
Comment (unknownbrackets) [inline android/AndroidManifest.xml]: I'm mostly sure we don't support this format?

-[Unknown]

Comment (xiushudongfang) [inline android/AndroidManifest.xml]: Oops, I am sorry . It don't support bin and isz  according to my test just now.

Comment (xiushudongfang) [inline android/AndroidManifest.xml]: Need I pull a request to remove this ? Or you remove it by yourself ?

**Comment by sum2012** [comment]
It seems that buildbot don't like this change

**Comment by hrydgard** [comment]
@sum2012 indeed. I will take a look soon.

**Pull request merged**